### DISCUSSION TITLE
Is it possible to create a standby from a volume snapshot?

### DISCUSSION THREAD

**Discussion by anthonator** [cloudnative-pg/cloudnative-pg#4649] [Q&A]
I'm trying to scale up a cluster and I keep getting stuck. The standby seems to try and restore the same set of WAL files over and over. It doesn't seem to ever progress.

Has anybody ever successfully created a standby when using volume snapshot backups?

**Comment by ajrpayne** [comment]
I have the same issue. I made a https://github.com/cloudnative-pg/cloudnative-pg/issues/4488
